test/hotspot/gtest/gc/shared/test_oopStorageSet.cpp

```diff
@@ -44,52 +45,54 @@ class OopStorageSetTest : public ::testing::Test {
     return count;
   }
 
-  template <uint count>
-  static void check_iterator(OopStorageSet::Iterator it,
```

Comment:
All the functions you changed are named `_iterator` and tested OopStorageSet::Iterator. Now the name is the same, but instead they test the Range facility. I think these functions should be renamed. Alternatively, we keep the tests for the OopStorageSet::Iterator and create a new set for the Range?